ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionsReader.scala

```diff
@@ -50,6 +51,10 @@ private[dao] final class TransactionsReader(
 
   private val logger = ContextualizedLogger.get(this.getClass)
 
+  // TransactionReader adds an Akka stream buffer at the end of all streaming queries.
+  // This significantly improves the performance of the transaction service.
+  private val outputStreamBufferSize = 128
```

Comment:
I would still prefer if we moved constants like this into a single file, but I'm OK with this for now.